src/main/perl/Filesystem.pm

```diff
@@ -178,14 +238,13 @@ sub update_fstab
         $this_app->verbose("A CAF::FileEditor instance was passed. ",
```

Comment:
Is this really a message to the end user? If it's to the developer then should be debug() level?